atlasdb-cassandra-integration-tests/build.gradle

```diff
@@ -10,6 +10,8 @@ dependencies {
     testImplementation 'org.apache.thrift:libthrift'
     testImplementation 'org.awaitility:awaitility'
     testImplementation 'org.slf4j:slf4j-api'
+    testImplementation 'org.apache.commons:commons-pool2'
```

Comment:
This is shadowed in atlas-cassandra, which meant using the client factory which implements something from this causes NoClassDef error, so I added it as a test implementation here.

Reply:
We can't use the client pool since that creates the keyspace!